docs/svsm.md

```diff
@@ -0,0 +1,135 @@
+# SVSM  Utility Script (Launch_SVSM.sh)
+
+This 'Launch_SVSM.sh'  utility script is a provisioning and test script that performs three tasks:
+
+1. Sets up an AMD EPYC CPU powered server by building host kernel, guest kernel, IGVM,  qemu, ovmf builds
```

Comment:
```suggestion
1. Sets up an AMD EPYC CPU powered server by building SVSM compatible host kernel, guest kernel, IGVM,  qemu and ovmf builds
```

Reply:
Agreed. Thanks for pointing that out; I'll update it.